app/lib/csv_importer.rb

```diff
@@ -1,5 +1,6 @@
 class CsvImporter
-  attr_reader :stats, :model, :filename, :processed_file_id
+  attr_reader :stats, :model, :filename, :processed_file_id, :erred
+  alias :erred? :erred
```

Comment:
How would you feel about changing this name to `errored`?  I agree that `erred` is more correct, but I wonder if `errored` might be more readable?

Reply:
I'll change it to `errorred`